src/MonoTorrent/MonoTorrent.Client.Tracker/UdpTracker.cs

```diff
@@ -60,6 +60,8 @@ protected override async Task<List<Peer>> DoAnnounceAsync (AnnounceParameters pa
                 var message = new AnnounceMessage (DateTime.Now.GetHashCode (), connectionId, parameters);
                 var announce = (AnnounceResponseMessage) await SendAndReceiveAsync (message);
                 MinUpdateInterval = announce.Interval;
+                Complete = announce.Seeders;
+                Incomplete = announce.Leechers;
```

Comment:
This would be good as a separate PR, ideally with a unit test. I can add one once the change is split up if you do not have the time to do so!